### PR TITLE
#17: Fix broken links

### LINKED ISSUE #17: Fix broken links reported by broken_links_checker.yml
ERROR: 4 dead links found!
[✖] https://opensource.org/licenses/MIT → Status: 503
[✖] http://opensource.org/licenses/BSD-3-Clause → Status: 503
[✖] https://www.mojohaus.org/flatten-maven-plugin/flatten-maven-plugin → Status: 404
[✖] http://anthonywhitford.com/lombok.maven/lombok-maven-plugin/" → Status: 404

Can probably be fixed by updating project-keeper to latest version which makes broken_links_checker.yml ignore links to opensource.org as they have blocked broken_links_checker.yml due too many requests.

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #17.

### CODE REVIEW

File: doc/changes/changes_2.1.1.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Error Code Model Java 2.1.1, released 2022-08-01
+
+Code name:
```
Comment (ckunki) [inline doc/changes/changes_2.1.1.md]:
```suggestion
Code name: Dependency update and fix for broken links
```

File: pk_generated_parent.pom [hunk 2]
```diff
@@ -3,28 +3,54 @@
...
+    <scm>
+        <connection>scm:git:https://github.com/exasol/error-code-model-java.git</connection>
+        <developerConnection>scm:git:https://github.com/exasol/error-code-model-java.git</developerConnection>
+        <url>https://github.com/exasol/error-code-model-java/</url>
+    </scm>
     <dependencies>
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.20</version>
```
Comment (ckunki) [inline pk_generated_parent.pom:18]: Do you want to remove lombok later?

**Review by ckunki — CHANGES_REQUESTED** [review]

File: doc/changes/changes_2.1.1.md (hunk 1 above)
Comment (redcatbear) [reply doc/changes/changes_2.1.1.md]: Will be done with #19.

File: pk_generated_parent.pom (hunk 2 above)
Comment (redcatbear) [reply pk_generated_parent.pom:18]: Will be done with #19.

**Review by ckunki — APPROVED** [review]

**Pull request merged**